### DISCUSSION TITLE
error while deploying FundMe Script.

### DISCUSSION THREAD

**Discussion by AVI-SHUKLA1110** [Cyfrin/foundry-full-course-cu#1236] [Q&A]
https://youtu.be/sas02qSFZ74?t=1541 
i was trying to deploy this script but after running it via the terminal , this is the error i am being thrown with .
<img width="625" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/88367072/1a025acd-1cb4-4b80-879b-638f736dbee1">

**Comment by ghost** [comment]
The problem is the uppercase **_c_** in the name of the cheatcode.
